src/components/02-components/offering-form/offering-form.tsx

```diff
@@ -184,402 +217,442 @@ export default function OfferingForm({
+                label="Preferred language(s)"
+                options={preferredLanguages}
+                selected={defaultPreferredLanguages}
+                sx={formFieldStyles}
+              />
+            )}
+            {requiredLanguages && (
+              <AutocompleteField
+                multiple
+                id="offering-required-languages"
+                name="offering-required-languages"
+                label="Required language(s)"
+                options={requiredLanguages}
+                selected={defaultRequiredLanguages}
+                sx={formFieldStyles}
+              />
+            )}
+            {requirements && (
+              <AutocompleteField
+                multiple
+                id="offering-requirements"
+                name="offering-requirements"
+                label="Preferred/Required Skills and Knowledge"
+                options={requirements}
+                selected={defaultRequirements}
+                sx={formFieldStyles}
+              />
+            )}
+            {activities && (
+              <AutocompleteField
+                multiple
+                required
+                id="offering-activities"
+                name="offering-activities"
+                label="Activities"
+                helptext=""
+                options={activities}
+                selected={defaultActivities}
+                sx={formFieldStyles}
 
-          {formSigners && (
-            <AutocompleteField
-              multiple
-              required
-              id="offering-form-signers"
-              name="offering-form-signers"
-              label="Form signer(s)"
-              options={formSigners}
-              selected={defaultFormSigners}
-              sx={formFieldStyles}
-            />
-          )}
-          {observers && (
-            <AutocompleteField
-              multiple
-              id="offering-observers"
-              name="offering-observers"
-              label="Observer(s)"
-              options={observers}
-              selected={defaultObservers}
-              sx={formFieldStyles}
-            />
-          )}
-          {preferredLanguages && (
-            <AutocompleteField
-              multiple
-              id="offering-preferred-languages"
-              name="offering-preferred-languages"
-              label="Preferred language(s)"
-              options={preferredLanguages}
-              selected={defaultPreferredLanguages}
-              sx={formFieldStyles}
-            />
-          )}
-          {requiredLanguages && (
-            <AutocompleteField
-              multiple
-              id="offering-required-languages"
-              name="offering-required-languages"
-              label="Required language(s)"
-              options={requiredLanguages}
-              selected={defaultRequiredLanguages}
+              />
+            )}
+          </Box>
+
+          <Box component="fieldset" sx={fieldSetStyles}>
```

Comment:
These don't appear to be collapsible, discussing need with client in main channel on Slack

Reply:
In a follow up PR this will be switched to a MUI accordion component